packages/venia-ui/lib/RootComponents/Category/category.js

```diff
@@ -36,9 +37,21 @@ const Category = props => {
         totalPages
     };
 
-    const [queryResult, queryApi] = useQuery(categoryQuery);
-    const { data, error, loading } = queryResult;
-    const { runQuery, setLoading } = queryApi;
+    // Run the category query immediately.
+    const { loading, error, data, networkStatus, refetch } = useQuery(
+        GET_CATEGORY,
+        {
+            variables: {
```

Comment:
Can we move this variables object to a memo so we don't have to create it each time we refetch later?

Reply:
Resolved by https://github.com/magento/pwa-studio/pull/1827/commits/6a1a0dfcfb2d81df22f96c274b2a90f35c7a197e.